fas2ipa/users.py

```diff
@@ -233,7 +233,11 @@ def _compact_value(cls, val):
     def migrate_user(self, person):
         person_orig = person
         person = person.copy()
-        if self.config["users"]["skip_disabled"] and person.get("status") != "active":
+        if (
+                self.config["users"]["skip_disabled"] 
+                and person.get("status") != "active" 
+                and person.get("status") != "bot"
```

Comment:
To make this more concise: `... and person.get("status") not in ("active", "bot") ...`

Reply:
Ok the latest commit should fix that